### PR TITLE
fixing up sidebar and rewording

### PR DESCRIPTION
**Pull request by mlautman**
[Live version](http://tutorials.picknik.ai/doc/move_group_interface/move_group_interface_tutorial.html)

### CODE REVIEW

File: doc/move_group_interface/move_group_interface_tutorial.rst [hunk 1]
```diff
@@ -50,9 +53,3 @@ Watch the `YouTube video demo <https://youtu.be/_5siHkFQPBQ>`_ for expected outp
  9. The object is removed from the environment.
 
 .. |B| image:: ./move_group_interface_tutorial_robot_with_box.png
-
-Explaining the Demo
-^^^^^^^^^^^^^^^^^^^
-The entire code is located in the moveit_tutorials github repo under the subfolder :codedir:`move_group_interface<move_group_interface>`. Next we step through the code piece by piece to explain its functionality.
-
-.. tutorial-formatter:: ./src/move_group_interface_tutorial.cpp
```
Comment (davetcoleman) [inline doc/move_group_interface/move_group_interface_tutorial.rst]: why remove this??

Comment (mlautman) [reply doc/move_group_interface/move_group_interface_tutorial.rst]: tutorial-formatter got moved up to line 17. This was done to standardize the tutorials

Comment (davetcoleman) [reply doc/move_group_interface/move_group_interface_tutorial.rst]: lets standardize to have the roslaunch demo *before* the explanation

Comment (mlautman) [reply doc/move_group_interface/move_group_interface_tutorial.rst]: Sounds good. Added to my todo for a separate PR

**Pull request merged**